src/main/views/features/help/accessibility-statement/accessibility-statement.njk

```diff
@@ -0,0 +1,150 @@
+{% extends "layouts/default.njk" %}
+
+{% block backLink %}
+  {# Remove back button on this page by replacing it with nothing #}
+{% endblock %}
+
+{% block main_content %}
+
+<main id="page-container" class="search govuk-width-container" role="main">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div id="search-container">
+        <article class="text" id="page-content">
+          <h1 class="govuk-heading-xl">
+            Accessibility statement for the Companies House service
```

Comment:
I can just double check if there is any requirement of having this page in welsh ?